float64[] v
---
float64[] q_dot
